mysql-ballerina/tests/json-type-test.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/test;
+
+string jsonTypeDB = "JSON_TYPE_DB";
+
+type JsonTable record {|
+    int id;
+    json json_value;
+|};
+
+type JsonTableValue record {
+    JsonTable value;
+};
+
+@test:Config {
+    groups: ["json-type"]
+}
+function testGetJsonType() {
+    Client dbClient = checkpanic new (host, user, password, jsonTypeDB, port);
+    stream<record{}, error> streamData =
```

Comment:
Isnt this query test case? Better add it to query-complex group. It is better to have limited groups

Reply:
I will refactor code. thanks for the input.